prow/config/inrepoconfig.go

```diff
@@ -88,7 +106,10 @@ func prowYAMLGetter(
 	if orgRepo.Repo == "" {
 		return nil, fmt.Errorf("didn't get two results when splitting repo identifier %q", identifier)
 	}
+
+	timeBeforeClone := time.Now()
 	repo, err := gc.ClientFor(orgRepo.Org, orgRepo.Repo)
+	inrepoconfigMetrics.gitCloneDuration.WithLabelValues(orgRepo.Repo).Observe((float64(time.Since(timeBeforeClone).Seconds())))
```

Comment:
Why not measure all the git operations up through line 148? If there is other git related slowness it would be nice to capture that with this metric.

Reply:
SGTM, will update (will add another separate metric because I am especially curious about the clone time).